test/basic.html

```diff
@@ -46,6 +46,10 @@
 
     <script>
 
+      function ensureDocumentHasFocus() {
```

Comment:
I tried each separately but neither alone seemed work. :/ I'll give it another shot to double check though.